packages/lisk-transactions/test/utils/validation/validation.ts

```
@@ -238,6 +238,16 @@ describe('validation', () => {
 			});
 		});
 
+		describe('Given an address that includes `.`', () => {
+			const address = '14.15133512790761431L';
+			const error =
+				'Address format does not match requirements. Address includes invalid character: `.`.';
+
+			it('should throw', () => {
```

maybe `should throw an error` sounds better